src/outbackcdx/Capture.java

```diff
@@ -243,6 +281,12 @@ public Object get(String field) {
                 return compressedoffset;
             case "filename":
                 return file;
+            case "originalLength":
+                return (originalLength != -1) ? originalLength : "-";
```

Comment:
Above in `fromCdxLine()` and `toString()` the value 0 is treated as "-".

Reply:
In this particular instance I was using -1 to signify some sort of null value, since a record could reasonably have an offset of 0, which is a valid value, I'm open to exploring this in another way if you have any suggestions?